### ISSUE TITLE
Report this site tool

### ISSUE THREAD

**Issue by JimKillock** [openrightsgroup/blocked-org-uk#74]
One a user has chosen to report a site to an ISP, they should be presented with a form to do so.

The form will submitted the data to ISPs email address; with a bcc to the user. A record sent to ORG or marked in the dabase..
- The Form should include a checkbox: 
  “I have checked this site (link) is not squatted, abandoned, a redirected URL or unsuitable for under 18s."

Once a report has been submitted
- direct to thank you for reporting, click here to check another site - link to random site.

**Comment by dantheta** [comment]
Added the checkbox and text.

**Comment by dantheta** [comment]
Thank you page spun off to separate ticket

**Comment by JimKillock** [comment]
Can we see the report tool yet?

**Comment by JimKillock** [comment]
Add active link to "I have checked that this site (http://houseinguanajuato.com) is not squatted, abandoned, a redirected URL or unsuitable for under 18s."

**Comment by JimKillock** [comment]
Can we make it clearer why it won't email an ISP because the check is too old, eg a tool tip exlanation "We were unable to recheck if this website at this ISP, so you cannot email them via this form at the moment."

**Comment by dantheta** [comment]
Active link done.

**Comment by dantheta** [comment]
I've added a tooltip to the checkbox with slightly different wording before and after a recheck.

**Comment by dantheta** [comment]
Ready for review, I think.

**Comment by JimKillock** [comment]
Do we want an opt in / out for the subsequent emails? Specifically the second part of this?

"Entering your details and site information will send an automated email from blocked.org.uk to the content administration contact for each of the selected networks. The blocked.org.uk platform will re-test the chosen URL each day, and will email you when the site is unblocked on any of the networks."

If you are bulk checking this could be an annoying feature!

**Comment by dantheta** [comment]
Yep, that's a good plan.  Added a checkbox (default off) to the form and updated the wording.

**Comment by JimKillock** [comment]
This looks good. In relation to the emails, when it is live, will the From: / Reply-to: headers match the original sender? 

Also, I think we want the sender to get a "confirm" email with a link so that ISPs can't claim they don't know who the originator is.

**Comment by pamcowburn** [comment]
In text email sent by Jim, the ampersand appears like this in email: B&amp;B.
:
This appears to be a winery and B&B. Although it is related to alcohol in a tangential sense, its primary purpose is to attract visitors to stay in the B&B as guests.

**Comment by dantheta** [comment]
@JimKillock Using the end-user's email address may give us problems with SPF and DKIM, causing the email to be dropped by anti-spam measures. I think there are other headers we can use to add authenticity though. Reply-to should be fine though.

**Comment by JimKillock** [comment]
The minimum requirement is that we validate the email address prior to sending so that ISPs can't claim the emails are not always truly from the sender. 

We should also send a copy to the originator so they can follow up as part of a thread if they don't get a response.

**Comment by pamcowburn** [comment]
'Send me email updates when this site is unblocked.' - think we should be clear that these are from ORG not the ISP

**Comment by dantheta** [comment]
Updated the text to show where the update emails come from.  I'll spin email validation into a separate ticket.
